tabbycat/api/urls.py

```diff
@@ -224,6 +224,14 @@
                         views.VenueCategoryViewSet.as_view(detail_methods),
                         name='api-venuecategory-detail'),
                 ])),
+                path('/actionlog', include([
+                    path('',
+                        views.ActionLogViewSet.as_view(list_methods),
+                        name='api-actionlog-list'),
+                    path('/<int:pk>',
+                         views.ActionLogViewSet.as_view(detail_methods),
```

Comment:
```suggestion
                         views.ActionLogViewSet.as_view({'get': 'retrieve'}),
```
Same here. This also prevents deletion.